Proposals/Glyph_Extension_Axis/ProposalSummary.md

```diff
@@ -0,0 +1,69 @@
+# Proposal Summary Form
+
+## Administrative Information
+**Proposers name:** Sahar Afshar and José Solé
+
+**Vendor affiliation:** N/A
+
+**Proposal name:** Glyph Extension axis
+
+**Date of submission:** 2 January 2018
+
+**New or revised proposal:** New
+
+**Previous revision date:** N/A
+
+## General Technical Information
+**Overview:** This axis applies a change to the width (or extension) of a single glyph, in addition to any global parameters previously set in place by other axes. This is especially useful for manipulating the length of words or sentences locally, as opposed to adjusting the overall width of the font.
+
+**Related axes:** This axis should take effect once all other axes have been applied.
+
+**Similar axes:** To clarify, the proposed axis is different from the width or extended/condensed axis; the Glyph Extension axis is intended for local use, an example is that in a condensed typeface, a single glyph can be made wider. In connecting scripts, this will only affect connecting strokes.
+
+**Axis type:** Parametric with optical option
+
+## Proposed Axis Details
+
+**Tag:** gext
+
+**Name:** Glyph Extension
+
+**Description:** Used to vary the extension of a glyph by a positive or negative value, depending on parameters assigned by the layout engine (when setting lines of text) or by optical adjustments made by the user (for single words or shorter lengths of text).
+
+This axis works on a single instance of a glyph, where it will apply a change on its extension (outline and spacing) on top of any other changes that have been previously applied via other axes.
+
+**Valid numeric range:** Any negative, zero or positive value.
+
+**Scale interpretation:** Starting with 0 as the glyph in its ‘normal’ state after all other axes have been applied, any increase or decrease should represent the delta applied to the advanced width in font units.
+
+**Recommended or required ‘Regular’ value:** 0
+
+**Suggested programmatic interactions:** When used to adjust the space of a line of justified text, applications should, taking into consideration the script's rules, adjust the extensions of individual glyphs to satisfy the desired line length.
+
+**UI recommendations:** UI recommendations change depending on how the axis is being used, which is itself dependant on environment, script and type of user.
+
+For Arabic the default should be to set this axis as hidden. Programmatic interaction for the Arabic script happens when a user sets a paragraph of text to be ‘fully justified’ or flush with both sides of the text box. The layout engine will then apply this axis on selected glyphs within a line of text to best fit the line to a specific length for justification purposes. A fully defined algorithm for this interaction should be defined, communicated and discussed with application developers for conventionality and interoperability purposes.
```

Comment:
>  The layout engine will then apply this axis on selected glyphs

Reply:
Will clarify this point. Selected is not the correct word.

Reply:
Cool!

Reply:
More generally, I'm curious why you wouldn't just select a character and apply a different regular width axis value to that. On the web, you'd need something like http://letteringjs.com to wrap the selected character in a span, and then apply a specific CSS variations property value to it.

Reply:
Because this axis doesn't effect the overall width of a character, it is intended to extend parts of a glyph. This is explained further in the axis proposal, see **similar axes**.

Reply:
Aha right! So, why wouldn't I just select a character and apply a different XTRA value to it?

Reply:
It is our understanding that XTRA and WDTH changes the overall character shape. The proposed GEXT axis only effects parts of a glyph. Excuse the crude illustration, but this is the best way I can show the difference between the two: 
![img_6193](https://user-images.githubusercontent.com/17496291/34533775-9a16303c-f0b3-11e7-842f-47a8f3d72d1a.jpg)
And this doesn't only apply to connecting scripts. As we explain in the proposal, in the 42-line Gutenberg Bible parts of letters are shortened to help with better word fit:  
![b42_smaletyper_220](https://user-images.githubusercontent.com/17496291/34534088-d0f47e32-f0b4-11e7-9a0d-f8a5b892e655.jpg)

Reply:
👌

Reply:
(@jmsole @sahafshar sorry about the mix up here, David's reply above is in regard to another discussion thread elsewhere on this repo.)